Source/WebCore/platform/graphics/gstreamer/mse/MediaPlayerPrivateGStreamerMSE.cpp

```diff
@@ -56,20 +56,46 @@
 #include <wtf/text/AtomStringHash.h>
 #include <wtf/text/StringToIntegerConversion.h>
 
+static void videoDecoderLimitsDefaults(unsigned& mediaMaxWidth, unsigned& mediaMaxHeight, unsigned& mediaMaxFrameRate)
```

Comment:
Better return a boolean indicating if parsing has been successful (true) or not (false).

Reply:
Even better to return a `std::optional<std::tuple<unsigned, unsigned, unsigned>>`.